vueapp/src/App.vue

```diff
@@ -231,7 +241,37 @@ export default {
 				this.disableEditMode();
 				this.setLocked(true);
 			}
+		},
+		duplicateToggleDarkMode(id) {
```

Comment:
Seems like these functions are unused. If they are needed sometime later down the line, can we add them later instead?

Reply:
Sure, ok then.